server/index.js

```diff
@@ -24,26 +25,10 @@ app.use('/test', (req, res) => {
   res.send('3-2-1 testing! Server is serving!');
 });
 
-app.get('/:id', (req, res) => {
-  if (req.params.id > 100 || req.params.id < 1) {
-    let errorMessage = 'Out of range error! Please choose a number 1 - 100.'
-    res.send(errorMessage);
-
-  } else {
-    res.sendFile('/Users/kimmybeee/Desktop/kimarie-bonuses/client/dist/index.html', (err) => {
-      if (err) {
-        res.send(err);
-      } else {
-        console.log('HTML re-served');
-      }
-    });
-  }
-});
-
-// To render items in Bonus Tier
-app.get('/bonus/:bundleId', (req, res) => {
-  let bundleId = req.params.bundleId;
-
+//Get request fired from within the HTML
+app.get('/bonus/:id', cors(), (req, res) => {
+  let bundleId = req.params.id;
+  console.log(bundleId);
```

Comment:
Are you using this for testing? If this has been working consistently, it may be worth having fewer `console.log`s. See also line 72.